source/priority-queue.ts

```diff
@@ -32,6 +32,27 @@ export default class PriorityQueue implements Queue<RunFunction, PriorityQueueOp
 		this.#queue.splice(index, 0, element);
 	}
 
+	setPriority(id: string, priority?: number) {
+		const existingIndex: number = this.#queue.findIndex((element: Readonly<PriorityQueueOptions>) => element.id === id);
+		const [item] = this.#queue.splice(existingIndex, 1);
+		if (item === undefined) {
+			return;
+		}
+
+		item.priority = priority ?? ((item.priority ?? 0) + 1);
```

Comment:
In my opinion, `.setPriority('abc')` does not make it obvious enough that it increases the priority by `1`. What about if one wanted to decrease priority. Maybe more methods might be needed?

Reply:
incrementing the priority by 1, might not be a generic use-case, so reverting it to be user specified.